www/static/img/ergol.svg

```diff
@@ -3,6 +3,25 @@
      viewBox="0 0 900 300">
 
   <style>
+    :root {
+        --color-palette-a: #9CF6F6;
+        --color-palette-b: #F3C98B;
+        --color-palette-c: #DAA588;
+        --color-palette-d: #C46D5E;
+        --color-palette-e: #F56960;
+
+        --color-palette-a-light: #DAFCFC;
+        --color-palette-b-light: #FBEEDA;
+        --color-palette-c-light: #F5E7E0;
+        --color-palette-d-light: #EED7D2;
+        --color-palette-e-light: #FBC9C5;
+        --color-palette-a-dark: #0E9595;
+        --color-palette-b-dark: #CB8215;
+        --color-palette-c-dark: #9C5630;
+        --color-palette-d-dark: #77372C;
+        --color-palette-e-dark: #99130A;
+    }
```

Comment:
I’d much prefer to keep the original variable names, but as many of these variables are unused I assume this is a temporary abstraction layer?